modules/rest-api/pages/rest-setting-security.adoc

```diff
@@ -0,0 +1,173 @@
+= Manage Encryption Settings
+:page-topic-type: reference
+
+[abstract]
+Establish and retrieve cluster-wide settings for the use of encryption and cipher-suites.
+
+== HTTP method and URI
+
+----
+GET /settings/security
+
+POST /settings/security
+----
+
+== Description
+
+Allows settings to be established for the use of encryption.
+This includes the encryption _level_, for node-to-node encryption within the cluster;
+UI disablement over insecure connections; TLS minimum version; and a list of server-accepted cipher suites.
+
+[#curl-syntax]
+== Curl Syntax
+
+----
+curl -X GET -d hostname=<host>:<port>
+  -d user=<administrator>
+  -d password=<password>
+  -u <administrator>:<password>
+
+curl -X POST -d hostname=<host>:<port>
+  -d user=<administrator>
+  -d password=<password>
+  -u <administrator>:<password>
+  -d disableUIOverHttp=<true|false>
+  -d disableUIOverHttps=<true|false>
+  -d clusterEncryptionLevel=<all|control>
+  -d tlsMinVersion=<tlsv1|tlsv1.1|tslv1.2>
+  -d honorCipherOrder=<true|false>
+  -d cipherSuites=<list-of-accepted-cipher-suites>
+  http://<host>:<port>/settings/security
+----
+
+The syntax for the `POST` method includes the following:
+
+* `disableUIOverHttp`.
+Whether access to Couchbase Web Console should be disabled over http.
+Default is `false`, meaning that the console can be accessed over http.
+
+* `disableUIOverHttps`.
+Whether access to Couchbase Web Console should be disabled over https.
+Default is `false`, meaning that the console can be accessed over https.
+
+* `clusterEncryptionLevel`.
+Controls the level of encryption imposed on inter-node communications within the cluster.
```

Comment:
It only works when node-to-node encryption is turned on. If it doesn't there is no encryption neither for control traffic, nor for data traffic

Reply:
Have added.